lib/exunit.ex

```diff
@@ -56,8 +54,8 @@ defmodule Testcontainers.ExUnit do
   defmacro container(name, config, options \\ []) do
     run_block =
       quote do
-        {:ok, container} = Container.run(unquote(config), on_exit: &ExUnit.Callbacks.on_exit/1)
-
+        {:ok, container} = Testcontainers.start_container(unquote(config))
+        ExUnit.Callbacks.on_exit(fn -> Testcontainers.stop_container(container.container_id) end)
```

Comment:
if the above start_container call, fails, for ex if if fails to wait for it to complete, it will be cleaned up by Ryuk